RELEASE.md

```diff
@@ -135,6 +136,10 @@
   * Use `tf.compat.v1.estimator.inputs` instead of `tf.estimator.inputs`
   * Replace `contrib` references with `tf.estimator.experimental.*` for APIs in `early_stopping.py`
   * Determining the “correct” value of the `--iterations_per_loop` for TPUEstimator or DistributionStrategy continues to be a challenge for our users. We propose dynamically tuning the `--iterations_per_loop` variable, specifically for using TPUEstimator in training mode, based on a user target TPU execution time. Users might specify a value such as: `--iterations_per_loop=300s`, which will result in roughly 300 seconds being spent on the TPU between host side operations.
+* TensorRT
+  * Migrate TensorRT conversion sources from contrib to compiler directory in preparation for TF 2.0.
+  * Add additional, user friendly `TrtGraphConverter` API for TensorRT conversion.
+  * Expand support for TensorFlow operators such as `CombinedNonMaxSuppression` in TensorRT conversion which significantly accelerates object detection models.
```

Comment:
This requires TRT 5.1, but 1.14 is still built with 5.0, do we want to make a note?

Reply:
They would have to recompile with  5.1 just installing it is not enough because these are new operators.  Correct?  I would just remove this, but unless someone else complains I think it is fine if we add a note that you have to recompile or install TRT 5.1 to get the gains, which ever one is what the use has to do.

Reply:
Yes it requires recompilation. Agree that adding a note seems better instead of removal.

Reply:
Good point. Note added.